packages/velo-external-db/test/drivers/schema_api_rest_test_support.js

```diff
@@ -9,6 +11,14 @@ const givenCollection = async(name, columns, auth) => {
     }
 }
 
+const retrieveSchemaOperations = async() => {
+    const tempCollectionName = 'tempCollection'
+    await axios.post('/schemas/create', { collectionName: tempCollectionName }, authOwner)
```

Comment:
why do we need to create a collection to retrieve schema capabilities, don't we have that even if we don't have any collections ?